dev_guide/managing_images.adoc

```diff
@@ -1105,6 +1105,85 @@ my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
 ----
 endif::openshift-origin,openshift-dedicated,openshift-online[]
 
+[[mirror-registry-images]]
+== Transferring Images
+
+To move one or more images from one Docker registry to another,
+use `oc image mirror` command. The images are streamed from registry to registry
+without being stored locally.
+
+For example, to copy image from dockerhub to the integrated registry use the
+following command:
+
+[source, bash]
+----
+$ oc image mirror docker.io/library/busybox:latest 172.30.124.220:5000/myproject/toybox:latest
+----
+
+[IMPORTANT]
+====
+If you use `docker.io` in source or destination you can't omit `docker.io` and
+`library` components. You can't omit `latest` if you want to get latest tag.
+====
+
+It's possible to copy an image to several locations at once. To do this, you
+need to specify several destinations:
+
+[source, bash]
+----
+$ oc image mirror 172.30.124.220:5000/myproject/busybox:latest docker.io/myrepository/busybox:stable docker.io/myrepository/toybox:dev
+----
+
+[NONE]
+====
+The `oc image mirror` runs locally, not inside the openshift cluster. This means
+that this command must have access to the source and destination registry.
+====
+
+If the Docker registry requires authentication to pull or push an image, you
+you need to log in manually (using the `docker login` command) before executing
+the oc image mirror command.
+
+[[mirror-registry-images-s3]]
+=== Mirroring Images to S3
```

Comment:
per discussion w/ @smarterclayton about this and the issues you ran into with it, let's remove the s3-related bits of this doc.

Reply:
@bparees done